pkg/controller/handler.go

```diff
@@ -8,20 +8,20 @@ import (
 
 // Handler is an implementation of a ResourceEventHandler.
 type Handler struct {
-	ignored               k8s.IgnoreWrapper
-	configRefreshChan     chan string
-	createMeshServiceFunc func(service *corev1.Service) error
-	updateMeshServiceFunc func(oldUserService *corev1.Service, newUserService *corev1.Service) (*corev1.Service, error)
-	deleteMeshServiceFunc func(serviceName, serviceNamespace string) error
+	ignored           k8s.IgnoreWrapper
+	configRefreshChan chan string
+	shadowSvc         ServiceManager
```

Comment:
```suggestion
	serviceManager         ServiceManager
```

Reply:
Don't you find `shadowSvc` a bit confusing? its not a service object..its a manager interface right?